src/sql/src/session/vars.rs

```diff
@@ -621,6 +621,30 @@ pub const ENABLE_RBAC_CHECKS: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+/// This is separate from the [`FORCE_INTROSPECTION_CLUSTER`] `ServerVar` so we can
+/// independently roll out this feature via LaunchDarkly without effecting user's ability
+/// to disable the behavior for there sessions.
+pub const ENABLE_FORCE_INTROSPECTION_CLUSTER: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("enable_force_introspection_cluster"),
+    value: &false,
+    description:
+        "Whether the feature to force queries that depends only on system tables to run on the mz_introspection cluster, is enabled (Materialize).",
+    internal: true,
+    safe: true,
+};
+
+/// This is separate from the [`ENABLE_FORCE_INTROSPECTION_CLUSTER`] `ServerVar` so we
+/// can independently roll out this feature via LaunchDarkly. Users can set this var as
+/// a session variable, while we can control the feature overall with the former.
+pub const FORCE_INTROSPECTION_CLUSTER: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("force_introspection_cluster"),
+    value: &true,
+    description:
+        "Whether to force queries that depend only on system tables, to run on the mz_introspection cluster (Materialize).",
+    internal: true,
```

Comment:
I think if users are meant to set this, then `internal` should be false? (double check me on this because I'm not sure).

Reply:
Totally, updated this, and the testdrive test that asserts the results of `SHOW ALL`.

Note: I'm going to hold off on adding it to any documentation yet, e.g. [`session-variables.html`](https://github.com/MaterializeInc/materialize/blob/main/doc/user/layouts/shortcodes/session-variables.html) until the feature is 100% released.